_releases/2023-06-20-may-2023.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+category: releases
+title: May 2023 Release Notes
+filename-custom: may-2023
+last_modified_at: 
+sidenav: about_us
+release-year: "2023"
+description: "Release notes for May 2023, including several improvements and one fix."
+---
+## Improvements
+
+* **Results Page Redesign**: We’ve continued progress on updating our frontend components.
```

Comment:
Another instance of smart quotes that I missed earlier.

Reply:
straight-quotes-ified